server/config/middleware.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var userRoutes = require('../users/userRoutes.js');
+var messageRoutes = require('../messages/messageRoutes.js');
+var contactRoutes = require('../contacts/contactRoutes.js');
+
+module.exports = function (app, express) {
+
+  // instantiate express Routers for User, Message, and Contact 
+  // see: http://expressjs.com/guide/routing.html
+  var userRouter = express.Router();
+  var messageRouter = express.Router();
+  var contactRouter = express.Router();
+
```

Comment:
Based on our conversation today (and updates to our Angular routes for '/'), is app.use(express.static(__dirname + '/../../client')) still necessary?